addons/config/run.tanzu.vmware.com/v1alpha3/kappcontrollerconfig.yaml

```diff
@@ -0,0 +1,24 @@
+#@ load("@ytt:data", "data")
+---
+apiVersion: run.tanzu.vmware.com/v1alpha3
+kind: KappControllerConfig
+metadata:
+  name: #@ data.values.TKR_VERSION
+  namespace: #@ data.values.GLOBAL_NAMESPACE
+spec:
+  namespace: test-ns
```

Comment:
Should be `data.values.GLOBAL_NAMESPACE`

Reply:
Fixed